src/app/login/page.tsx

```diff
@@ -39,48 +91,127 @@ const LoginPage = () => {
         login(response.token);
         router.push('/');
       }
-    } catch (error) {
-      setError('Špatný email nebo heslo.');
+    } catch (error: any) {
+      if (error.response.status === 400) {
+        setError('Špatně zadané přihlašovací údaje.');
+      } else {
+        setError('Něco se pokazilo. Zkuste to prosím znovu později.');
+      }
+      setIsLoading(false);
     }
   };
 
+  const isDisabled = Boolean(emailError || passwordError || passwordError);
+
   return (
-    <div className="flex flex-col items-center justify-center">
-      <form
-        className="mt-32 flex flex-col border border-black p-4 md:w-1/3"
-        onSubmit={handleSubmit}
-      >
-        <label className="mb-4 w-full">
-          Email:
+    <section className="flex flex-col items-center justify-center pt-20">
+      <h1 className="text-xl font-semibold">
+        Přihlášení se do <span className="text-primary">SportujSpolu</span>
+      </h1>
+      <p className="mt-4">Chci se přihlásit přes:</p>
+      <ul className="my-5 flex">
+        <li>
+          <button onClick={() => window.alert('🚀 Feature coming soon! 🌟')}>
+            <FacebookIcon className="mx-3 transform transition-transform duration-200 ease-in-out hover:scale-115 hover:fill-primary" />
+          </button>
+        </li>
+        <li>
+          <button onClick={() => window.alert('🚀 Feature coming soon! 🌟')}>
+            <GoogleIcon className="mx-3 transform transition-transform duration-200 ease-in-out hover:scale-115 hover:fill-primary" />
```

Comment:
Anything that is hover, should be probably `focus-within` as well. Please find a difference between `focus` and `focus-outline`. Lemme know on Slack the difference. I have my thoughts but I am not sure.

Right now, there is no visible focus 
![CleanShot 2023-09-27 at 15 41 01](https://github.com/GLObus303/sportujspolu/assets/12488648/c9b7d225-8dc2-4a85-8c5d-20d87a13530b)